contrib/golang/filters/http/source/go/pkg/http/shim.go

```diff
@@ -45,13 +45,22 @@ var ErrDupRequestKey = errors.New("dup request key")
 
 var Requests = &requestMap{}
 
+var envoyConcurrency uint32
+
+// EnvoyConcurrency returns the concurrency Envoy was set to run at. This can be used to optimize HTTP filters that need
+// memory per worker thread to avoid locks.
```

Comment:
better to add a note like this?
please do not use it in the Golang `init()` method, otherwise, it will return 0.
since the value is initilized when a Golang parsing configuration.